### DISCUSSION TITLE
Is the Carbon Hack Event taking place at a physical location?

### DISCUSSION THREAD

**Discussion by tofols** [Green-Software-Foundation/hack#120] [Ask Anything]
I would like some clarity if the sessions are done at physical venues.

If yes, how can I register to attend the physical event?
Thanks!

**Comment by russelltrow** [comment]
Hi @tofols Carbon Hack 24 is a virtual event.

Read the [Participant Guide](https://github.com/Green-Software-Foundation/hack/wiki/Participant-Guide) to understand how to get involved.
